jbx/recipes/kinesis.rb

```diff
@@ -19,6 +19,7 @@
           },
         }
       )
+      triggers_reload !node[:container]
```

Comment:
Shouldn't this become 
```suggestion
      triggers_reload  unless node[:container]
```

Reply:
Not in this case, `unless` is a conditional. We need to provide a boolean value to `triggers_reload`